common/src/main/java/com/microsoft/identity/common/internal/ui/webview/AzureActiveDirectoryWebViewClient.java

```diff
@@ -85,8 +86,9 @@ public AzureActiveDirectoryWebViewClient(@NonNull final Activity activity,
                                              @NonNull final String redirectUrl) {
         super(activity, completionCallback, pageLoadedCallback);
         mRedirectUrl = redirectUrl;
-        //Creating ClientCertAuthChallengeHandler starts YubiKitManager usb discovery
-        mClientCertAuthChallengeHandler = new ClientCertAuthChallengeHandler(getActivity());
+        //Creating ClientCertAuthChallengeHandler starts smartcard usb discovery
+        mClientCertAuthChallengeHandler = new ClientCertAuthChallengeHandler(getActivity(),
+                SmartcardCertBasedAuthManagerFactory.getSmartcardCertBasedAuthManager(getActivity()));
```

Comment:
Nit on naming

`get` here in a static context connotes a Singleton like functionality imo.... if it's not breaking, I might suggest a factory method name that implies object construction like `createSmartcardCertBasedAuthManager()`

Reply:
Renamed to `createSmartcardCertBasedAuthManager()`